### ISSUE TITLE
Folders of playlist

### ISSUE THREAD

**Issue by Sytten** [tomahawk-player/tomahawk#374]
Hi,

I'm coming from an itune setup and I want to completely move to tomahawk, but I miss the feature to class my playlist in a directory like tree (with folders, sub-folders, etc.)
Would that be hard to implement?
I have some experience in C++ and Qt so I could probably do it, but its a big software without a real documentation and I dont know where to start...

Cheers

**Comment by gordielachance** [comment]
would like to see that too !

**Comment by dschmidt** [comment]
Ping @muesli

**Comment by nicholsjer** [comment]
Yes please - my 50+ playlists are a bit much to scroll through when all contained at the same level
